From ec88fc43aa96bedae1ad32d07e0237fe6fb940b1 Mon Sep 17 00:00:00 2001
From: fsiegert <frank.siegert@cern.ch>
Date: Wed, 4 Apr 2018 10:10:55 +0200
Subject: [PATCH] Fix compilation when LIBAVFORMAT_VERSION_MICRO==100

Since UBFFmpegVideoEncoder.cpp uses `avresample.h` for `LIBAVFORMAT_VERSION_MICRO<100` we have to be complementary here in the include file when including `swresample.h`.
---
 src/podcast/ffmpeg/UBFFmpegVideoEncoder.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/podcast/ffmpeg/UBFFmpegVideoEncoder.h b/src/podcast/ffmpeg/UBFFmpegVideoEncoder.h
index 6068a369..69fa81ef 100644
--- a/src/podcast/ffmpeg/UBFFmpegVideoEncoder.h
+++ b/src/podcast/ffmpeg/UBFFmpegVideoEncoder.h
@@ -36,7 +36,7 @@ extern "C" {
     #include <libswscale/swscale.h>
 
 // Due to the whole ffmpeg / libAV silliness, we have to support libavresample on some platforms
-#if LIBAVFORMAT_VERSION_MICRO > 100
+#if LIBAVFORMAT_VERSION_MICRO >= 100
     #include <libswresample/swresample.h>
 #else
     #include <libavresample/avresample.h>
